README.md

```diff
@@ -18,21 +23,194 @@ scheme. The SDK's contract is determined by the top-level exports from
+
+const HelloWorldWorkflowDef = DefineWorkflow({
+  callback_id: "hello_world_workflow",
+  title: "Hello World Workflow",
+  input_parameters: {
+    properties: {
+      channel: {
+        type: Schema.slack.types.channel_id,
+      },
+    },
+    required: ["channel"],
+  },
+});
+
+const helloWorldStep = HelloWorldWorkflowDef.addStep(HelloWorldFunctionDef, {});
+
+HelloWorldWorkflowDef.addStep(Schema.slack.functions.SendMessage, {
+  channel_id: HelloWorldWorkflowDef.inputs.channel,
+  message: helloWorldStep.outputs.message,
+});
+
+export default HelloWorldWorkflowDef;
+```
+
+`DefineWorkflow` is used to define a workflow and provide Slack with the
+information required to use it.
+
+`HelloWorldWorkflow.addStep` is used to add a step to the workflow, here we add
+the `HelloWorldFunction` and then the `SendMessage` Slack Function that will
+post the `message` to a Slack channel.
+
+### Update the [manifest](https://api.slack.com/automation/manifest)
+
+```ts
+// Contents of manifest.ts
+import { Manifest } from "deno-slack-sdk/mod.ts";
+import HelloWorldWorkflow from "./workflows/hello_world.ts";
+
+export default Manifest({
+  name: "my-app",
+  description: "A Hello World app",
+  icon: "assets/default_new_app_icon.png",
+  workflows: [HelloWorldWorkflow],
+  outgoingDomains: [],
+  botScopes: ["chat:write", "chat:write.public"],
+});
+```
+
+`Manifest` is used to define your apps
+[manifest](https://api.slack.com/automation/manifest) and provides Slack with
+the information required to manage it.
+
+### Create a [trigger](https://api.slack.com/automation/triggers)
+
+```zsh
+mkdir ./triggers && touch ./triggers/hello_world.ts
+```
+
+```ts
+// Contents of ./triggers/hello_world.ts
+import { Trigger } from "deno-slack-sdk/types.ts";
+import { TriggerContextData, TriggerTypes } from "deno-slack-api/mod.ts";
+import HelloWorldWorkflow from "../workflows/hello_world.ts";
+
+const trigger: Trigger<typeof HelloWorldWorkflow.definition> = {
+  type: TriggerTypes.Shortcut,
+  name: "Reverse a string",
+  description: "Starts the workflow to reverse a string",
+  workflow: `#/workflows/${HelloWorldWorkflow.definition.callback_id}`,
+  inputs: {
+    channel: {
+      value: TriggerContextData.Shortcut.channel_id,
+    },
+  },
+};
+
+export default trigger;
+```
+
+The `Trigger` object is used to define a trigger that will invoke the
+`HelloWorldWorkflow`. The Slack CLI will detect this file and prompt you for its
+creation.
+
+## Running an app
+
+```zsh
+slack run
+```
+
+When prompted, create the `triggers/hello_world.ts` trigger! This will send your
+trigger configuration to Slack.
 
-If you make changes to this repo, or just want to make sure things are working
-as desired, you can run:
+Post the `Hello world shortcut trigger` in a slack message and **use it**
 
-    deno task test
+## Getting Help
 
-To get a full test coverage report, run:
+[This documentation](https://api.slack.com/automation) has more information on
```

Comment:
```suggestion
[This documentation](https://api.slack.com/automation/deno/develop) has more information on
```

Reply:
Great catch 💯  I will add a link to https://api.slack.com/automation/deno/develop along side the link to https://api.slack.com/automation